designs/waralee/src/ghost.mjs

```diff
@@ -0,0 +1,107 @@
+import { pantsProto } from './pantsproto.mjs'
+
+export const ghost = {
+  name: 'waralee.ghost',
+  after: pantsProto,
+  draft: ({
+    options,
+    measurements,
+    Point,
+    Path,
+    points,
+    paths,
+    Snippet,
+    snippets,
+    sa,
+    macro,
+    part,
+  }) => {
+    const c = 0.55191502449351
+    const eyeSize = 0.02
```

Comment:
This file should not be in this PR, as it's part of Gozer.

However, it does confirm my feeling that something went wrong with the eye size in Gozer.
The way you have set up `size` and `eyeSize` here, this code will function as expected.
But in Gozer, the values of `size` and `eyeSize` are derived from measurements, with `eyeSize` specically being `measurements.head * 0.0416` so the code needs to be adapted.

But also, please remove this file from this PR ;)